payloads/v10/channel.ts

```diff
@@ -914,6 +914,10 @@ export enum MessageFlags {
 	 * This message has a snapshot (via Message Forwarding)
 	 */
 	HasSnapshot = 1 << 14,
+	/**
+	 * This flag is required to use new components
```

Comment:
You may want to consider adding documentation around what this flag actually does (additional components, restrictions on other fields, etc). Additionally, these components wont be "new" forever, so improved documentation may be in order

Reply:
Currently descriptions are temporarily copied from the alpha readme - once the docs pr is out with proper descriptions we'll use those